### DISCUSSION TITLE
[Document Bug]Invalid script in japanese document

### DISCUSSION THREAD

**Discussion by no-yan** [community/community#17950] [Other Feature Feedback, Questions, & Ideas]
I found an invalid code in GitHub Doc. 

In this [link](https://docs.github.com/ja/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases#auto-launching-ssh-agent-on-git-for-windows), code offered for windows doesn't work because by reasons below:
- Two functions of same name
- One of them is not properly enclosed in parentheses

```sh
agent_start () {
    (umask 077; ssh-agent >| "$env")
    . "$env" >| /dev/null ; }
agent_start () {
    (umask 077; ssh-agent >| "$env")
    . "$SSH_AUTH_SOCK" ] || [ $agent_run_state = 2 ]; then
    agent_start
    ssh-add
elif [ "$SSH_AUTH_SOCK" ] && [ $agent_run_state = 1 ]; then
    ssh-add
fi
```

[The English version](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases#auto-launching-ssh-agent-on-git-for-windows) has an executable code and seemingly the code above is made in translation.
Please correct it to be the same as the English one.

For clarity, I copied all the code in problem.

```sh
env=~/.ssh/agent.env

agent_load_env () { test -f "$env" && . "$env" >| /dev/null ; }

agent_start () {
    (umask 077; ssh-agent >| "$env")
    . "$env" >| /dev/null ; }

agent_load_env

# agent_run_state: 0=agent running w/ key; 1=agent w/o key; 2=agent not running
agent_run_state=$(ssh-add -l >| /dev/null 2>&1; echo $?)

"$env" >| /dev/null ; }

agent_start () {
    (umask 077; ssh-agent >| "$env")
    . "$SSH_AUTH_SOCK" ] || [ $agent_run_state = 2 ]; then
    agent_start
    ssh-add
elif [ "$SSH_AUTH_SOCK" ] && [ $agent_run_state = 1 ]; then
    ssh-add
fi
```
https://docs.github.com/ja/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases

**Comment by queenofcorgis** [comment]
Hey @no-yan - we appreciate you flagging this! I added the proper label to this so the appropriate team can review.

**Reply by no-yan** [reply]
About two months have passed and there may be no progress.

This is a no big deal Issue, it would just be finished in 10 minutes.
On the other hand, for someone new to SSH connections on GitHub, the right documentation would be a huge help.
I hope the translation team realizes this issue...

@liligalante Do you know who to mention to get things moving forward?
Or should I repost this elsewhere appropriate?

**Comment by no-yan** [comment]
Resolved. Thank you for addressing the issue.
